README.md

```diff
@@ -26,43 +27,44 @@ Steel flexi-daggers with a rubber blunt, or a nylon trainer. Additional control
 Rapier Fencing Equipment
 
 Armour:
-1. Helmet (12-kilogram three-weapon/sabre fencing mask or better) with back of head protection.
+1. Helmet (12-kilogram three-weapon/sabre fencing mask or better). No cuts to the back of the head are allowed unless the helmet includes back of the head protection.
```

Comment:
This is my best guess regarding the "If a fencing mask is in use, no cuts are allowed to the back of the head." rule below.